### ISSUE TITLE
Unable to set ThreatDetectionPolicy on SQL Database

### ISSUE THREAD

**Issue by dfibuch** [pulumi/pulumi-azure-nextgen#257]
I am using latest version of `@pulumi/azure-nextgen` v0.5.1 and trying to set the `DatabaseArgs` like this:

```TS
    const defaultArgs: azure.sql.v20200801preview.DatabaseArgs = {
        databaseName,
        location: locationName,
        resourceGroupName: groupName,
        serverName,
        threatDetectionPolicy: {
            state: "Enabled"
        }
    };
```

However `threatDetectionPolicy` is not exposed and I cannot find a way to set it. It is supported in `@pulumi/azure`. Do I have to access some other API to do this or is it just not supported at this moment?

EDIT: Here is the sample from Microsoft for the ARM template that accepts this setting https://github.com/Azure/azure-quickstart-templates/blob/master/201-sql-threat-detection-db-policy-multiple-databases/azuredeploy.json#L85

**Comment by mikhailshilkov** [comment]
Yeah, it looks like the API shape of `securityAlertPolicies` requires some manual intervention to map it to Pulumi API, so it's missing right now. It will be a separate resource when we expose it.

**Comment by dfibuch** [comment]
Thank you for the quick response @mikhailshilkov . Not going to hold you to it but any idea of timescale to get it in? Just trying to decide if I should use `@pulumi/azure` for the SQL templates.

**Comment by mikhailshilkov** [comment]
I have it working locally, it seems. I'll run the tests and ship it, if so.

**Comment by mikhailshilkov** [comment]
The `DatabaseSecurityAlertPolicy` resource will be released in 0.6.0
